service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java

```diff
@@ -250,10 +247,13 @@ private Workflow sendWorkflowRequest(WorkflowDto workflow, String requestPath, S
       throws WorkflowEngineServiceException {
 
     HttpEntity<WorkflowDto> entity = new HttpEntity<>(workflow, headers(tenant, token));
-    String url = String.format(requestPath, okapiUrl, basePath);
+    HttpMethod method = HttpMethod.POST;
+    String url = String.format(requestPath, okapiUrl);
+
+    log.info("Send Okapi workflow engine request {} {}", method, url);
```

Comment:
```suggestion
    log.debug("Send Okapi workflow engine request {} {}", method, url);
```
Might be better as `debug`.

Reply:
Done.